### PR TITLE
cockpit-components-logs-panel: React-related fixes

### PR DESCRIPTION
**Pull request by mareklibra**
Part of #9263 PR-split, see individual commits for more info.

### CODE REVIEW

**Comment by martinpitt** [comment]
Looks ok to me (aside from inconsistent quoting, but this is not important), but this needs a rebase against current master. Thanks!

**Comment by mareklibra** [comment]
Rebased. Inconsistent quoting fixed.

File: pkg/lib/cockpit-components-logs-panel.jsx [hunk 1]
```diff
@@ -44,8 +44,10 @@ class JournalOutput {
             warning = true;
         }
 
+        // the element will be member of a generated array
+        const key = { ident, prio, message, count, time };
         return (
-            <div className="cockpit-logline" role="row">
+            <div className="cockpit-logline" role="row" key={key}>
```
Comment (martinpitt) [inline pkg/lib/cockpit-components-logs-panel.jsx]: As we've found out in other PRs, using objects as key apparently doesn't work.

**Comment by mvollmer** [comment]
I pushed some proposed fixes.

**Comment by mvollmer** [comment]
I squashed the commits and force pushed.

**Comment by martinpitt** [comment]
For some strange reason this causes ovirt build to fail:
```
chunk    {1} ovirt/vnc.min.js, ovirt/vnc.min.js.map (ovirt/vnc) 344 kB [rendered]

ERROR in ./pkg/ovirt/ovirt.less
Module build failed: 

      &:hover {
        background: @navbar-pf-item-hover-bg-color;
                  ^
Variable @navbar-pf-item-hover-bg-color is undefined
      in /tmp/source/node_modules/patternfly-react/dist/less/masthead.less (line 13, column 20)
 @ multi ovirt/ovirt
Child extract-text-webpack-plugin:
```

**Comment by martinpitt** [comment]
I pushed a rebase, and the mysterious ovirt failure in semaphore is gone now (semaphore doesn't rebase, but our integration tests do). So this is good now, waiting for tests.

**Pull request merged**